infrastructure-components.yaml

```diff
@@ -0,0 +1,5464 @@
+                                                        items:
+                                                          type: string
+                                                        type: array
+                                                    required:
+                                                    - key
+                                                    - operator
+                                                    type: object
+                                                  type: array
+                                                matchLabels:
+                                                  additionalProperties:
+                                                    type: string
+                                                  description: matchLabels is a map of {key,value} pairs. A single {key,value} in the matchLabels map is equivalent to an element of matchExpressions, whose key field is "key", the operator is "In", and the values array contains only "value". The requirements are ANDed.
+                                                  type: object
+                                              type: object
+                                            storageClassName:
+                                              description: 'Name of the StorageClass required by the claim. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1'
+                                              type: string
+                                            volumeMode:
+                                              description: volumeMode defines what type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec.
+                                              type: string
+                                            volumeName:
+                                              description: VolumeName is the binding reference to the PersistentVolume backing this claim.
+                                              type: string
+                                          type: object
+                                      type: object
+                                    status:
+                                      description: DataVolumeTemplateDummyStatus is here simply for backwards compatibility with a previous API.
+                                      nullable: true
+                                      type: object
+                                  required:
+                                  - spec
+                                  type: object
+                                type: array
+                              flavor:
+                                description: FlavorMatcher references a flavor that is used to fill fields in Template
+                                properties:
+                                  kind:
+                                    description: 'Kind specifies which flavor resource i
```

Comment:
this image needs to point to a specific image associated with the tag this infrastructure component yaml is generated from. Otherwise this image tag will continue to change in the future which will result in a mismatch of this yaml and the container image.

Reply:
Fixed